lib/glific/partners.ex

```diff
@@ -1011,7 +1011,7 @@ defmodule Glific.Partners do
 
         {:error, error} ->
           Logger.info(
-            "Error fetching token for: #{provider_shortcode}, error: #{error}, org_id: #{organization_id}"
+            "Error fetching token for: #{provider_shortcode}, error: #{inspect(error)}, org_id: #{organization_id}"
```

Comment:
since the error is `account not found` we should disable GCS also by marking `is_active` flag as false and create notification as well with severity as `critical`